watertap/data/techno_economic/waiv.yaml

```diff
@@ -2,29 +2,26 @@ default:
   energy_electric_flow_vol_inlet:
     value: 0.65
     units: kWh/m^3
-  specific_capital_cost_flow_mass_inlet:
-    flow_basis_par:
+  capital_cost:
+    basis: flow_mass
+    cost_factor: None
+    reference_state:
       value: 64.0
       units: kg/hr
-    cap_basis_par:
-      value: 0.11
-      units: M$
-    cap_exp_par:
+    capital_a_parameter:
+      value: 1.1e5
+      units: USD_2020
+    capital_b_parameter:
       value: 1.0
       units: dimensionless
-    mass units:
-      value: kg/hr
-    capital cost units:
-      value: M$
-    year:
-      value: 2020
-    cost_function_form:
-      function f(flow_mass_in): cap_basis_par*(flow_mass_in/flow_basis_par)^cap_exp
-      units: $
   recovery_frac_mass_H2O:
-    value: 0.9999
+    value: 0.0001
```

Comment:
Just to confirm is this change correct?

Reply:
I would say this is correct and good catch by @luohezhiming. In WT3 the WAIV was always a terminal unit so we didn't care about the recovery because the flow went nowhere. But this process is intended to have most of the inflow be evaporated so would in reality have very little water recovery